Sources/SwiftGenKit/Stencil/ResourcesContext.swift

```diff
@@ -0,0 +1,26 @@
+//
+// SwiftGenKit
+// Copyright © 2019 SwiftGen
+// MIT Licence
+//
+
+import Foundation
+
+extension Resource.Parser {
+  public func stencilContext() -> [String: Any] {
+    let files = self.files
+      .sorted { lhs, rhs in lhs.name < rhs.name }
+      .map(map(file:))
+
+    return [
+      "files": files
+    ]
+  }
+
+  private func map(file: Resource.File) -> [String: Any] {
+    return [
+      "name": file.name,
```

Comment:
I think it could be useful to not only have the "name" but also the "path" (relative path).
That way, if a developer adds a folder as a "Folder reference" to their project – leading to the directory structure to be kept and copied as-in in the final app bundle – they can still access it using that `path` (while if it's included in the folder as a normal file it will end up at the root of the bundle's resource directory and users would use `name` instead)

Reply:
👍 for paths. Note that the relative path calculation actually happens here: https://github.com/SwiftGen/SwiftGen/blob/stable/Sources/SwiftGenKit/Parsers/Fonts/CTFont.swift#L22

We can easily add mime-type info as well, or some sort of "readable" type information. In case the template wants to do some special case handling for some specific types.